tests/e2e/pageobjects/login/openshift/RedHatLoginPage.ts

```diff
@@ -34,6 +36,18 @@ export class RedHatLoginPage {
 		await this.driverHelper.waitVisibility(RedHatLoginPage.USERNAME_INPUT, TIMEOUT_CONSTANTS.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
 	}
 
+	async waitAndConfirmCookiePolicy(): Promise<void> {
+		Logger.debug();
+
+		await this.driverHelper.waitVisibility(RedHatLoginPage.COOKIE_DIALOG_IFRAME, TIMEOUT_CONSTANTS.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
+		await this.driverHelper
+			.getDriver()
+			.switchTo()
+			.frame(await this.driverHelper.getDriver().findElement(RedHatLoginPage.COOKIE_DIALOG_IFRAME));
+		await this.driverHelper.waitAndClick(RedHatLoginPage.COOKIE_DIALOG_BUTTON, TIMEOUT_CONSTANTS.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
+		await this.driverHelper.refreshPage();
```

Comment:
Refreshing the page is unusual step, and could hide another bug.
Is it really needed?

Reply:
I was having issues using `webdriver.switchTo().originalContent()` as forums suggested, it would time out and refresh anyways.
This seems to be really stable though.

Reply:
Got it.